scripts/cloud.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+source .envrc
+
+# Generate report of current env vars
+echo "======================================================"
+printenv | grep "KKA_.*"
+echo "======================================================"
+
+# Install ArgoCD
+kubectl create namespace argocd
+helm repo add argo https://argoproj.github.io/argo-helm
+helm repo update
+helm install argocd argo/argo-cd --namespace=argocd -f ./addons/argocd/values.yaml
```

Comment:
Thanks @fabiolrodriguez for the refactor, I just realised that if we do this we will install the latest version of argoCD while we use a different version in the test.
Can we instead use the same argocd we use in [titl](https://github.com/nearform/k8s-kurated-addons/blob/main/Tiltfile#L83-L88) thanks.